setup.py

```diff
@@ -13,24 +13,51 @@
 with open(os.path.join(BASE_DIR, "deflate.c"), "r") as src:
     version = re.match(r'.*#define MODULE_VERSION "(.*?)"', src.read(), re.S).group(1)
 
-cpu = "arm" if platform.machine().startswith("arm") else "x86"
+prefer_system_libdeflate = not bool(os.environ.get("USE_BUNDLED_DEFLATE"))
+system_prefix_libdeflate = os.environ.get("LIBDEFLATE_PREFIX")
+
+try:
+    import pkgconfig as pc
+except ImportError:
+    print("Warning: can not import pkgconfig python package.")
```

Comment:
Do we need this warning if using the bundled version?

Reply:
No, in when forcing the bundled code, no pkgconfig is used.

Reply:
no warning any more if not used anyway.